### DISCUSSION TITLE
Why is loading or refreshing from any source of page together ?

### DISCUSSION THREAD

**Discussion by focgcbog111** [zauberzeug/nicegui#491] [Q&A] [answered]
Hi i have small question, why when deploy or host my code in any server for example i use digital-ocean droplet server, everything its fine and  my code working well but when i access the page from any browser or any smart-phone browser, for example if i refresh the page from my smart-phone browser the open page in my laptop also refreshing ! .....also if i click any button or anything other pages work together in same time ?     i hope my question is clear .

**Comment by rodja** [comment, marked as answer]
This was already asked/answered in #461. In short: What you see is due to the default shared page. Use the [ui.page decorator](https://nicegui.io/reference#auto-index_page) for true private pages.

**Reply by focgcbog111** [reply]
Thanks for your answer
